java/dds/OpenDDS_DCPS_jni.cpp

```diff
@@ -135,6 +157,43 @@ jstring JNICALL Java_OpenDDS_DCPS_TheServiceParticipant_get_1unique_1id
   return retStr;
 }
 
+jobject JNICALL Java_OpenDDS_DCPS_TheServiceParticipant_network_1config_1modifier
+(JNIEnv * jni, jclass)
+{
+  try {
+    OpenDDS::DCPS::NetworkConfigModifier_rch ncm(TheServiceParticipant->network_config_modifier(), OpenDDS::DCPS::inc_count());
+    if (ncm == 0) {
+      return 0;
+    } else {
+      jclass ncmClazz = findClass(jni, "OpenDDS/DCPS/NetworkConfigModifier");
+      jmethodID ctor = jni->GetMethodID(ncmClazz, "<init>", "(J)V");
+      return jni->NewObject(ncmClazz, ctor, reinterpret_cast<jlong>(ncm._retn()));
+    }
+  } catch (const CORBA::SystemException &se) {
+    throw_java_exception(jni, se);
+    return 0;
+  }
+}
+
+// NetworkConfigModifier
+
+// NetworkConfigModifier::_jni_fini
+void JNICALL Java_OpenDDS_DCPS_NetworkConfigModifier__1jni_1fini
+(JNIEnv * jni, jobject jthis)
+{
+  OpenDDS::DCPS::NetworkConfigModifier_rch ncm = OpenDDS::DCPS::rchandle_from(recoverCppObj<OpenDDS::DCPS::NetworkConfigModifier>(jni, jthis));
+  ncm->_remove_ref();
+}
+
+// NetworkConfigModifier::update_interfaces
+void JNICALL Java_OpenDDS_DCPS_NetworkConfigModifier_update_1interfaces
+(JNIEnv * jni, jobject jthis)
+{
+  OpenDDS::DCPS::NetworkConfigModifier* ncm = recoverCppObj<OpenDDS::DCPS::NetworkConfigModifier>(jni, jthis);
+  ncm->update_interfaces();
+}
```

Comment:
References to `NetworkConfigModifier` are [causing errors](https://github.com/iguessthislldo/OpenDDS-Android/runs/1647110278). These should probably be NOPs if  `OPENDDS_NETWORK_CONFIG_MODIFIER` isn't defined.